domains/sxi.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "Sxinar",
+           "email": "130384567+Sxinar@users.noreply.github.com",
```

Comment:
You can't use a noreply email

Reply:
If you don't want your email to be exposed, you can remove the remove the email, but your discord should be there.

Reply:
@Stef-00012 oshidev@proton.me

Reply:
you need to edit the json file and replace the email with that by the way

Reply:
> you need to edit the json file and replace the email with that btw

Reply:
@CuteDog5695 update now.

Reply:
why cant you?

Reply:
> neden yapamıyorsun?

I am Turkish and I updated the necessary file, only my English is poor.

Reply:
fine, ill update the file for you